packages/node/test/unit/edge-functions/edge-matcher.test.ts

```diff
@@ -63,6 +63,12 @@ describe('middleware matchers', () => {
     expect(buildResult.routes).toEqual([
       {
         src: regExps.join('|'),
+        middlewareRawSrc:
+          matcher === undefined
+            ? []
+            : Array.isArray(matcher)
+            ? matcher
+            : [matcher],
```

Comment:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be.  -->
**suggestion (non-blocking):** extract or refactor

I find ternary expressions to be confusing, especially when nested. I suggest we extract this or even write a small helper here like:

```
function toArray(obj) {
  if (!obj) {
    return [];
  }

  if (Array.isArray(obj)) {
    return obj;
  }

  return [obj];
```

then:

```
middlewareRawSrc: toArray(matcher)
```